src/Serialization/Address.purs

```diff
@@ -376,12 +376,12 @@ derive instance Generic NetworkId _
 instance Show NetworkId where
   show = genericShow
 
-pubKeyAddress :: NetworkId -> Ed25519KeyHash -> BaseAddress
-pubKeyAddress netId pkh = baseAddress
+pubKeyAddress :: NetworkId -> Ed25519KeyHash -> Ed25519KeyHash -> BaseAddress
```

Comment:
The names of the arguments make it clear when reading the source, but it might also be nice to indicate what the two `Ed25519KeyHash`s are for in a comment